test/test_quantization.py

```diff
@@ -0,0 +1,273 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+import torch.nn.quantized as nnq
+import torch.quantization as tq
+from torch.quantization import *
+from common_utils import TestCase, run_tests
+
```

Comment:
please also add a test that explicitly inserts QuantStub/DequantStub and doesn't call add_quant_dequant. This is to make sure that we don't force everyone into that API

Reply:
added a test